### ISSUE TITLE
[false-positive] replace "file_put_contents()" with "copy()"

### ISSUE THREAD

**Issue by voku** [kalessil/phpinspectionsea#187]
It seems that if there is an class with the same method-name ("file_get_contents"), there will be a false-positive.

```php
file_put_contents(
  $file,
  UTF8::file_get_contents($file)
);
```
